### DISCUSSION TITLE
Creating a chained property (shortcut)

### DISCUSSION THREAD

**Discussion by mquantin** [geovistory/.github#14] [Q&A] [answered]
Hello, 
I'm trying to create a shortcut property for local convenient use of the CIDOC-CRM 
Property in ontoMe: https://ontome.net/property/2372/namespace/261

![schema of the property shortcut](https://svgur.com/s/122h)

Beside the scote note, is there a way to declare it in ontoMe as a chained property with owl statements? 
in turtle owl it would be: 
'''turtle
crms:P1_has_conceived  owl:propertyChainAxiom
   ( crm:P94_has_created  crm:P33_used_specific_technique crm:P31_has_modified ).
'''

**Comment by joschne** [comment, marked as answer]
Hi @mquantin 

Unfortunately the link of "schema of the property shortcut" is broken.

There is an ongoing discussion about [property paths](https://github.com/orgs/geovistory/discussions/1). How you see your question in relation to this?

**Comment by mquantin** [comment]
It is exactly the same topic. 
I post there.
